_contentTemplates/chart/link-to-basics.md

```diff
@@ -186,7 +186,8 @@ This approach is not limited only to the Labels - it can be used with to all tag
 #end
 
 #context-parameter-information
-* `Value` - maps to the value of the data point that is being hovered.
+* `FormattedValue` - maps to the value of the data point that is being hovered, formatted as a string.
```

Comment:
Maybe add a sentence here that it contains the default rendering of the tooltip, rather than the value. For example, in more complex charts (such as OHLC) it will probably be an actual `<table>`.